mbuild/compound.py

```diff
@@ -1024,16 +1083,7 @@ def save(self, filename, show_ports=False, forcefield_name=None,
 
         structure = self.to_parmed(**kwargs)
 
-        if box is None:
-            box = self._gen_box()
-        write_hoomdxml(structure, filename, forcefield, box,
-                       [id for id in self.rigid_ids()], **kwargs)
-
-    def save_gsd(self, filename, structure, forcefield_name,
-                 forcefield_files, box=None, **kwargs):
-        """ """
-        from mbuild.formats.gsdwriter import write_gsd
-        forcefield = False
+        # Apply a force field with foyer if specified
         if forcefield_name or forcefield_files:
             from foyer import Forcefield
```

Comment:
I guess we should talk about making foyer a required vs optional dependency but that's a separate issue